docs/RFCS/range_replica_naming.md

```diff
@@ -0,0 +1,41 @@
+- Feature Name: range_replica_naming
+- Status: draft
+- Start Date: 2015-07-29
+- RFC PR:
+- Cockroach Issue:
+
+# Summary
+
+Consistently use the word **range** to refer to a portion of the global
+sorted map (i.e. an entire raft consensus group, and **replica** to
+refer to a single copy of the range's data.
+
+# Motivation
+
+We currently use the word "range" informally to refer to both the
+consensus group and the data owned by one member of the group, while
```

Comment:
Is there a term we need to use for the the consensus group itself, or are you making it a part of Range itself?